kebechet/managers/version/utils.py

```diff
@@ -205,11 +204,29 @@ def _is_merge_event(payload: Dict[str, Any]):
             "This function only works for GitHub payloads at this time."
         )
     if (
-        payload["event"] == "pull_request"
-        and payload["raw_payload"]["payload"]["action"] == "closed"
+            payload["event"] == "pull_request"
+            and payload["raw_payload"]["payload"]["action"] == "closed"
     ):
         return payload["raw_payload"]["payload"]["pull_request"]["merged"]
 
 
 def _pr_id_from_webhook(payload: Dict[str, Any]):
     return payload["raw_payload"]["payload"]["number"]
+
+
+def _is_version_pr(payload: Dict[str, Any]):
+    return "version" in payload["raw_payload"]["payload"]["pull_request"]["title"]
```

Comment:
why is this separate?
in the function `_is_release_pr `, can we do "Release of version" ?

Reply:
Yes, please condense into a single function. It should just check branch name and the PR author.

Reply:
Ack! thanks